src/emulator.rs

```diff
@@ -230,6 +230,15 @@ impl Emulator {
                     println!("eipsw: 0x{:08x}", self.virtual_boy.cpu.reg_eipsw());
                     println!("ecr: 0x{:08x}", self.virtual_boy.cpu.reg_ecr());
                 }
+                Ok(Command::ShowCpuCache) => {
+                    println!("CPU Instruction Cached enabled: {}", self.virtual_boy.cpu.chcw_cache_enabled);
+                    let (hits, misses) = self.virtual_boy.cpu.cpu_cache.get_stats();
+                    let percent_hit = (hits as f32 / (hits + misses) as f32) * 100.0;
```

Comment:
nit: `f64` is the recommended default floating point type to use, rather than `f32`, especially in non-perf-critical bits of code like this.